src/main/resources/org/craftercms/studio/api/v2/dal/ItemDAO.xml

```diff
@@ -1186,6 +1186,9 @@
             i.state = (i.state | #{onStatesBitMap}) &amp; ~#{offStatesBitMap},
             i.preview_url = REPLACE(i.preview_url, #{oldPreviewUrl}, #{newPreviewUrl}),
             i.previous_path = IFNULL(i.previous_path, i.path)
+        <if test="parentId != null">
```

Comment:
What if parent ID is null? Does that create a data issue silently?

Reply:
parent id is null only for site content root